bicep/azure-pipeline.yml

```diff
@@ -0,0 +1,30 @@
+trigger:
+  branches:
+    include:
+      - main
+  paths:
+    exclude:
+    - README.md
+
+variables:
+- name: buildConfiguration
+  value: Release
+
+name: 1.0$(Rev:.r)
+
+stages:
+
+  - stage: Build
+    jobs:
+    - template: deploy/templates/build.yml
```

Comment:
Should the yaml templates be in the deploy folder?

Reply:
I can see how it is related to building and deploying, so it makes sense in that regard.

I was just thinking of when the folder is made into an artifact. I can't see why the templates should be in the artifact :)